NGitLab.Tests/EnvironmentsTests.cs

```diff
@@ -84,15 +85,19 @@ public async Task Edit()
         // Validate newEnvNameToEdit is present
         Assert.That(envClient.All.FirstOrDefault(e => string.Equals(e.Name, newEnvNameToEdit, StringComparison.Ordinal)), Is.Not.Null);
 
+        // Handling for old version 15.x
+        var isVersion15 = context.IsGitLabVersionInRange(VersionRange.Parse("[15.0,16.0)"), out _);
```

Comment:
As it's repeated in multiple places in tests, how about a little method like the existing `IsGitLabVersionInRange`:

```csharp
// In the GitLabTestContext
public bool IsGitLabMajorversion(int major)
{
    var currentVersion = Client.Version.Get();
    return NuGetVersion.TryParse(gitLabVersion, out var nuGetVersion) && nuGetVersion.Major == major;
}

// In tests, that would look like
if (context.IsGitLabMajorVersion(15)) 
{ 
    // Additional asserts
}
```

Reply:
Done